src/reducers/authReducer.ts

```diff
@@ -11,14 +12,24 @@ type authReducerState = {
 
 const UPDATE_LAST_ACTIVE_AT = 'AUTH_REDUCER_UPDATE_LAST_ACTIVE_AT';
 
-export const updateLastActiveAt = {
-  type: UPDATE_LAST_ACTIVE_AT
+export const updateLastActiveAt = (lastActiveAt: DateTime) => {
```

Comment:
Making `updateLastActiveAt` a function follows the pattern of `setUserGroups`, below, and means that we always pass in values to the reducer instead of having it create them using globals. This removes the need to mock out globals in tests (which was causing some issues after I upgraded some libraries).